src/components/Contacts/ContactDetails/ContactDetailsTab/ContactDetailsTab.tsx

```diff
@@ -65,90 +118,190 @@ export const ContactDetailsTab: React.FC<ContactDetailTabProps> = ({
   });
 
   const { t } = useTranslation();
+  const { enqueueSnackbar } = useSnackbar();
+  const { push, query } = useRouter();
+  const [deleteContact, { loading: deleting }] = useDeleteContactMutation();
+  const [dialogOpen, changeDialogOpen] = useState(false);
```

Comment:
This seems like a very general name for a very specific modal.

Reply:
Haha true, I can update it.